CIP-0XXX/README.md

```diff
@@ -0,0 +1,561 @@
+
+> [!WARNING]
+> Some compression can be applied to the Mithril signatures which allows them to always be on the lower bound size, but it is not implemented yet.
+
+> [!NOTE]  
+> For a total of **3,100** Cardano SPOs on the `mainnet`, on an average **50%** of them will be eligible to send signatures (i.e. will win at least one lottery in the Mithril protocol). This means that if the full Cardano stake distribution is involved in the Mithril protocol, only **1550** signers will send signatures at each round. 
+
+The following tables gather figures about expected network load in the case of **Mithril** using the mini-protocol to diffuse the individual signatures: 
+
+| Message part | Lower bound | Upper bound |
+|--------|------|------|
+| messageId | 32 B | 32 B | 
+| messageBody | 360 B | 2,000 B |
+| blockNumber | 4 B | 4 B |
+| kesSignature | 448 B | 448 B |
+| operationalCertificate | 304 B | 304 Bytes |
+
+| Message | Lower bound | Upper bound |
+|--------|------|------|
+| total | 1,148 B | 2,788 B | 
+
+For a total of **3,100** Cardano SPOs, there will be **1,550** messages sent per round of signature: 
+
+- the network outbound throughput of a peer is:
+
+| Send period | Lower bound | Upper bound |
+| ----------- | ----------- | ----------- |
+| 1 min       | 57 kB/s     | 138 kB/s    |
+| 2 min       | 29 kB/s     | 69 kB/s     |
+| 5 min       | 12 kB/s     | 28 kB/s     |
+| 10 min      | 6 kB/s      | 14 kB/s     |
+
+- the network outbound volume of a peer is:
+
+| Send period | Lower bound  | Upper bound  |
+| ----------- | ------------ | ------------ |
+| 1 min       | 147 GB/month | 356 GB/month |
+| 2 min       | 74 GB/month  | 178 GB/month |
+| 5 min       | 30 GB/month  | 72 GB/month  |
+| 10 min      | 15 GB/month  | 36 GB/month  |
+
+### Infrastructure extra operating costs
+
+#### Networking traffic cost
+
+> [!NOTE]  
+> - These data apply to cloud providers which bill the traffic on the volume, not the bandwidth.
+> - Some cloud providers offer a free tier for the first **100GB** of traffic which is not taken into consideration here for simplicity.
+
+| Cloud Provider | Inbound Traffic | Outbound Traffic |
+|--------|------|------|
+| AWS | 0 $/GB | 0.09 $/GB |
+| GCP | 0 $/GB | 0.11 $/GB |
+| Azure | 0 $/GB | 0.09 $/GB |
+| Average | 0 $/GB | 0.1 $/GB |
+
+
+#### Mithril message diffusion extra networking cost
+
+For a total of `3,000` SPOs sending messages, the extra networking cost incurred for a Cardano full node is:
+
+| Send period | Lower bound | Upper bound |
+| ----------- | ----------- | ----------- |
+| 1 min       | 15 $/month  | 36 $/month  |
+| 2 min       | 8 $/month   | 18 $/month  |
+| 5 min       | 3 $/month   | 8 $/month   |
+| 10 min      | 2 $/month   | 4 $/month   |
+
+### Protocol authentication
+
+#### Message authentication mechanism
+The message body is signed with the KES key of the SPO. This signature and the operational certificate of the SPO are appended to the message which is diffused.
```

Comment:
Can we move this further up and/or mention this in the introduction of the protocol? The fact that the whole protocol only works for signed messages that can be associated to stake on Cardano is very important.

Reply:
I have moved the section higher in the document.